nephio-ansible-install/roles/docker/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+## © 2022 Nokia
+## Licensed under the Apache License 2.0
+## SPDX-License-Identifier: Apache-2.0
+
+---
+- name: ensure repository key is installed
+  become: true
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+
+- name: ensure docker registry is available
+  become: true
+  apt_repository: repo='deb https://download.docker.com/linux/ubuntu bionic stable' state=present
+
+- name: ensure docker and dependencies are installed
+  become: true
+  apt: name=docker-ce update_cache=yes
+
+- name: Add user to docker group
+  become: true
+  shell: usermod -aG docker {{ cloud_user }}
+
+- name: Reset connection so docker group is picked up.
+  meta: reset_connection
+- name: Create the docker network for kind (to make our job easier plumbing up the lot)
+  docker_network:
```

Comment:
What pip3 module needs to be installed on the ansible controller to use this? I see the below error

```
vjayaram@wim-pr-test:~/nephio-ansible-install$ ansible-playbook -i 10.10.10.8, playbooks/install-prereq.yaml 
ERROR! couldn't resolve module/action 'docker_network'. This often indicates a misspelling, missing collection, or incorrect module path.

The error appears to be in '/home/vjayaram/nephio-ansible-install/roles/docker/tasks/main.yaml': line 27, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Create the docker network for kind (to make our job easier plumbing up the lot)
  ^ here
```